monitor.go

```diff
@@ -163,24 +170,30 @@ func (p *stageMonitor) draw(dc drawContext) {
 	default:
 		x, y := int(p.x), int(p.y)
 		labelRender := gdi.NewTextRender(defaultFont2, 0x80000, 0)
+		labelRender.Scale = p.size
```

Comment:
There are two different implementations for `TextRender` here:

1. `internal/gdi/gdi.go`, which will be used for build without `-tags canvas`
2. `internal/gdi/gdi_canvas.go`, which will be used for build with `-tags canvas`

In this PR the field `Scale` is added in the first implementation. So if we run

```sh
GOOS=js GOARCH=wasm go build -tags canvas
```

to build spx, it throws error:

```
# github.com/goplus/spx
./monitor.go:173:15: labelRender.Scale undefined (type gdi.TextRender has no field or method Scale)
./monitor.go:178:14: textRender.Scale undefined (type gdi.TextRender has no field or method Scale)
```

In web page of Builder, we use spx version which is built with `-tags canvas` to render within HTML Canvas.